airbyte-ci/connectors/pipelines/pipelines/dagger/actions/python/common.py

```diff
@@ -242,16 +242,16 @@ async def apply_python_development_overrides(context: ConnectorContext, connecto
     if context.use_local_cdk:
         context.logger.info("Using local CDK")
         # mount the local cdk
-        path_to_cdk = "airbyte-cdk/python/"
+        path_to_cdk = "../airbyte-python-cdk"  # Assume CDK is cloned in a sibling dir to `airbyte`
         directory_to_mount = context.get_repo_dir(path_to_cdk)
```

Comment:
`get_repo_dir` relates to the `airbyte` repo.
To get a directory on the host in the native dagger way you can do like this:

```suggestion
        directory_to_mount = context.dagger_client.host().directory(path_to_cdk)
```

(the relative path might not work, but an absolute path will for sure)